### ISSUE TITLE
Add support for CSS Level 4 outputs

### ISSUE THREAD

**Issue by elchininet** [elchininet/ColorTranslator#114]
The library [is prepared to accept CSS Level 4 inputs](https://github.com/elchininet/ColorTranslator#input) but it is not prepared to do the same with the output. At the moment, the output is always in CSS Level 3 and with a fixed format that [could not satisfy all the uses cases](https://github.com/elchininet/ColorTranslator/pull/113).

In this issue will be recorded all the work needed to make it possible to otput CSS colors in CSS Level 4 and different options to make the library more flexible with the output (the task list will be updated accordingly with each pull request).

- [x] Remove the decimal property from the class contructor and from the static methods and change it by a configuration object with the same option as a property (`{ decimals }`)
_by default it should be `MAX_DECIMALS`_ https://github.com/elchininet/ColorTranslator/pull/116
- [x] Remove complexity from the static methods creating new static methods (e.g. istead of `toHSL(color: ColorInput, css: boolean`, a method named `toHSL(color: ColorInput)` and a method named `toHSLObject(color: ColorInput)`) https://github.com/elchininet/ColorTranslator/pull/118
- [x] Add a new option to the configuration object to output the CSS in legacy mode (`{ legacy }`)
_by default it should be false_ https://github.com/elchininet/ColorTranslator/pull/119
- [x] Add a new option to the configuration object to use spaces after commas: (`{ spacesAfterCommas }`)
_by default it should be false_ https://github.com/elchininet/ColorTranslator/pull/120
- [x] Add a new option to the configuration object to set the angles units (`{ anglesUnit }`)
_by default it should be `none`. If it is `none` it will not have a unit but it defaults to `deg`. Possible enum values: `none`, `deg`, `grad`, `rad` and `turn` (this option only affects `hsl` css color output)_ https://github.com/elchininet/ColorTranslator/pull/121
- [x] Add a new option to the configuration object to set the rgb units (`{ rgbUnit }`)
_by default it should be `none`. Possible values `none` and `percent` (this option only affects `rgb` css color output)_ https://github.com/elchininet/ColorTranslator/pull/122
- [x] Move the logic for CSS config options to the CSS module https://github.com/elchininet/ColorTranslator/pull/123
- [x] Add a new option to the configuration object to set the cmyk units (`{ cmykUnit }`)
_by default it should be `percent`. Possible values `none` and `percent` (this option only affects `cmyk` css color output)_ https://github.com/elchininet/ColorTranslator/pull/124
- [x] Add a new option to the configuration object to set the alpha units (`{ alphaUnit }`)
_by default it should be `none`. Possible values `none` and `percent` (this option only affects `CSS Level 4` css color output)_ https://github.com/elchininet/ColorTranslator/pull/125
- [x] Add a new option to the configuration object to set the cmyk function (`{ cmykFunction }`)
_by default it should be `device-cmyk`. Possible values `device-cmyk` and `cmyk` (this option only affects `cmyk` css color output)_ https://github.com/elchininet/ColorTranslator/pull/126
- [x] Provide code examples of the new options in the documentation. https://github.com/elchininet/ColorTranslator/pull/127

**Comment by elchininet** [comment]
Hi @jeronimoek,
I‘ll start with the first task, I‘ll update it when the pull request is ready. Let me know in this issue when you are going to work in one so I can know it in advance.
Regards

**Comment by jeronimoek** [comment]
Hi @elchininet
After some research, I found another library that would help me translate to other formats. Due to that and other personal decisions, I decided that It would be more time effective for my goal to wrap that library and use it to solve the issue that triggered my original issue here. I'm really sorry for the inconvenience. For the moment, I might not be able to dedicate time to this issue.
Regards
